pypesto/history/base.py

```diff
@@ -249,7 +249,7 @@ def get_time_trace(
         trim: bool = False,
     ) -> Union[Sequence[float], float]:
         """
-        Cumulative execution times.
+        Cumulative execution times [s].
```

Comment:
Since pypesto already reports these in seconds, wouldn't it be better to convert the newly added quantities to seconds instead of ms?